resources/js/Molecules/Card.tsx

```diff
@@ -3,9 +3,10 @@ import React from "react";
 interface IProps {
     heading?: String;
     hasPadding?: Boolean;
+    children: React.ReactNode
```

Comment:
You don't have to add `children` to `IProps` if you don't want to - typescript knows from the React type definitions that all React components can have a `children` prop.

It is optional though (i.e. it's `children?: React.ReactNode` rather than `children: React.ReactNode`) so you could leave this here if you want to specify that the `Card` component **always** has to have children passed in to it.

Reply:
Interesting, thanks! Have removed it now.